logistics/index.md

```diff
@@ -6,3 +6,8 @@ has_children: true
 ---
 
 # Course Logistics
+
+## Quick Links
+
+- [CS 161 Pedagogy Blog](https://pedagogy.cs161.org/) 
```

Comment:
I'm honestly not super sure where the 161 pedagogy blog goes because it touches on so many things aside from course logistics, but I think it's probably fine to leave here for now.